docs/content/tutorial/advanced-tutorial/repositories.mdx

```diff
@@ -60,17 +42,17 @@ Now you can see the list of all jobs in the repo on the left:
 <Image
 alt="repos.png"
 src="/images/tutorial/repos.png"
-width={4018}
-height={3062}
+width={3808}
+height={2414}
```

Comment:
this screenshot is not showing the right thing as per https://dagster.slack.com/archives/G01JPESLH1C/p1641855376003000 - happy to regenerate it but not sure how